docs/models.rst

```diff
@@ -192,6 +192,21 @@ The ``Meta`` class
 
         Set this to configure a manual table name, instead of a generated one
 
+    .. attribute:: table_name_generator
+        :annotation: = None
+
+        Set this to a callable that will generate the table name for the model.
+        The callable should accept the model class as its argument and return a string.
+        If both ``table`` and ``table_name_generator`` are specified, ``table`` takes precedence.
```

Comment:
That seems confusing to me, and it can lead to confusing behaviour 

May be it would be better to raise configuration error in case both are specified?

Reply:
I thought about that, but imagine if you have an abstract model that all other models inherit from it, and for some table you want to use a custom name, how would you achieve this?

Reply:
I may be missing something, but I am pretty sure meta is not inherited 

```
from tortoise import Tortoise, fields, run_async
from tortoise.models import Model


class TournamentAbs(Model):
    id = fields.IntField(primary_key=True)
    name = fields.TextField()

    events: fields.ReverseRelation["Event"]

    class Meta:
        abstract = True
        table = "tournament1"
        table_description = "ttt"

    def __str__(self):
        return self.name


class Tournament(TournamentAbs):
    pass


async def run():
    await Tortoise.init(db_url="sqlite://:memory:", modules={"models": ["__main__"]})
    await Tortoise.generate_schemas()

    print(Tournament._meta.db_table)
    print(Tournament._meta.table_description or "None")


if __name__ == "__main__":
    run_async(run())
```

Gives output

```
tournament
None
```

If it was inherited - than `abstract` attribute would be inherited too